barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/client_ivc_recursive_verifier.cpp

```diff
@@ -0,0 +1,19 @@
+#include "client_ivc_recursive_verifier.hpp"
+
+namespace bb::stdlib::recursion::honk {
+
+void ClientIvcRecursiveVerifier_::verify(const ClientIVC::Proof& proof, VerifierInput& data)
```

Comment:
I think we settle on either IVC or Ivc when it comes to naming

Reply:
updated to IVC